pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
I like the idea of pickle - it will be bloody fast.
But be carefull with portability (saving in python2, loading in python3).

What can be considered is numpy save method: https://docs.scipy.org/doc/numpy/reference/generated/numpy.save.html

But it will only save cube data, not any other attribute.